app/lib/pdf_filler/az140_pdf.rb

```diff
@@ -24,9 +24,9 @@ def hash_for_pdf
         "1f" => @xml_document.at('Secondary TaxpayerSSN')&.text,
         "2a" => @xml_document.at("USAddress AddressLine1Txt")&.text,
         "2c" => @xml_document.at("USPhone")&.text,
-        "City, Town, Post Office" => @xml_document.at("CityNm")&.text,
-        "State" => @xml_document.at("StateAbbreviationCd")&.text,
-        "ZIP Code" => @xml_document.at("ZIPCd")&.text,
+        "City, Town, Post Office" => @xml_document.at("USAddress CityNm")&.text,
+        "State" => @xml_document.at("USAddress StateAbbreviationCd")&.text,
+        "ZIP Code" => @xml_document.at("USAddress ZIPCd")&.text,
```

Comment:
while making sure `StateAbbreviationCd` does not get pulled from W2, made sure to pull from USAddress (in ReturnHeader xml) like the rest of the address fields